### PR TITLE
KevS-SK1717-Update-SSL-Gateway-for-SSLGW

### PR DESCRIPTION
**Pull request by Kazarus4**
FRFR done.

Need peer and tech review.

### CODE REVIEW

File: pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md [hunk 1]
```diff
@@ -1,64 +1,68 @@
...
-### Prérequis
+L'offre SSL Gateway est conçue pour vous faire bénéficier d'un certificat SSL pour votre nom de domaine et votre service d'hébergement (VPS, serveur mail, serveur dédié, etc.).
 
-> [!primary]
+> [!warning]
 >
-> - Disposer d'un domaine ou sous-domaine actif.
-> - Disposer d'un accès à la zone DNS de celui-ci.
-> 
+> Les SSL Gateway sont incompatibles avec les offres d'[hébergement mutualisés OVHcloud](/links/web/hosting). Si vous souhaitez bénéficier d'un certificat SSL pour ce type d'offre, consultez notre guide « [Gérer un certificat SSL sur son hébergement web](/pages/web_cloud/web_hosting/ssl_on_webhosting) ».
```
Comment (Y0Coss) [inline pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md]:
```suggestion
> Les SSL Gateways sont incompatibles avec les offres d'[hébergements mutualisés OVHcloud](/links/web/hosting). Si vous souhaitez bénéficier d'un certificat SSL pour ce type d'offre, consultez notre guide « [Gérer un certificat SSL sur son hébergement web](/pages/web_cloud/web_hosting/ssl_on_webhosting) ».
```

File: pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md [hunk 2]
```diff
@@ -1,64 +1,68 @@
...
+>
+
+**Découvrer comment commander un SSL Gateway.**
```
Comment (Y0Coss) [inline pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md]:
```suggestion
**Découvrez comment commander un SSL Gateway.**
```

File: pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md [hunk 3]
```diff
@@ -1,64 +1,68 @@
...
 > [!warning]
 >
-> Offres Advanced et Enterprise :
-> 
-> Les domaines de 4ème niveau (blog.france.example.org) et plus sont autorisés.
-> 
-
-Notre système va ensuite détecter automatiquement la ou les adresses IP de votre site configurées sur votre domaine ou sous-domaine.
+> - Si vous disposez de plusieurs adresses IP pour votre site web, vous pourrez n'en choisir qu'une seule lors de la commande.
+> Si vous disposez de l'offre Advanced, vous pourrez par la suite ajouter jusqu'à 2 IPs supplémentaires depuis votre [espace client OVHcloud](/links/manager).
```
Comment (Y0Coss) [inline pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md]:
```suggestion
> - Si vous disposez de l'offre Advanced, vous pourrez par la suite ajouter jusqu'à 2 IPs supplémentaires depuis votre [espace client OVHcloud](/links/manager).
```

File: pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md [hunk 4]
```diff
@@ -1,64 +1,68 @@
...
+
 > [!warning]
 >
-> Offres Advanced et Enterprise :
-> 
-> Les domaines de 4ème niveau (blog.france.example.org) et plus sont autorisés.
-> 
-
-Notre système va ensuite détecter automatiquement la ou les adresses IP de votre site configurées sur votre domaine ou sous-domaine.
+> - Si vous disposez de plusieurs adresses IP pour votre site web, vous pourrez n'en choisir qu'une seule lors de la commande.
```
Comment (Y0Coss) [inline pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md]:
```suggestion
> - Si vous disposez de plusieurs adresses IP pour votre site web, vous ne pourrez choisir qu'une seule IP lors de la commande.
```

File: pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md [hunk 5]
```diff
@@ -1,64 +1,68 @@
...
+> Si vous disposez de l'offre Advanced, vous pourrez par la suite ajouter jusqu'à 2 IPs supplémentaires depuis votre [espace client OVHcloud](/links/manager).
+>
 
-- Si vous disposez de plusieurs IPs, sélectionnez en une.
-- Choisissez la zone géographique de votre service SSL Gateway
-- Si la zone DNS de ce domaine ou sous-domaine est géré par votre compte OVHcloud, cochez la case correspondante afin que nous modifions automatiquement la configuration de celui-ci pour pointer vers votre SSL Gateway.
-- Cliquez sur `Commander`{.action}.
+Ensuite, choisissez la localisation du datacentre où vous souhaitez installer le SSL Gateway parmi les 2 disponibles.
```
Comment (Y0Coss) [inline pages/web_cloud/ssl_gateway/order-ssl-gateway/guide.fr-fr.md]:
```suggestion
Choisissez ensuite la localisation du datacentre où vous souhaitez installer le SSL Gateway, parmi les 2 disponibles.
```

**Comment by Kazarus4** [comment]
Antoine B. confirmed by chat that all is technically good for him.

**Pull request merged**